def oops():
    raise Exception()

def doomed():
    try:
        oops()
    except IndexError:
        print('caught an index error!')
    except Exception as data:
        print('caught error:', MyError, data)
    else:
        print('no error caught...')

if __name__ == '__main__':
    doomed()

